.gitignore

```diff
@@ -6,6 +6,8 @@ __pycache__/
 # C extensions
 *.so
 
+solutions
```

Comment:
Why are we ignoring this folder?

Reply:
Because I have the solutions of Guille and i don't want to upload it to Github in order to avoid loading unnecessary data

Reply:
ok!